temporal-sdk/src/main/java/io/temporal/internal/statemachines/CancelNexusOperationStateMachine.java

```diff
@@ -74,6 +97,17 @@ private void createCancelNexusCommand() {
   }
 
   private void notifyCompleted() {
-    setInitialCommandEventId();
+    Failure canceledFailure =
```

Comment:
Why are we passing a failure back on `notifyCompleted`? If you look at `CancelExternalStateMachine` which is basically the same state machine but for workflows we don't pass anything back